### PR TITLE
Fix page buttons

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -** 
Fixes PageButtons props.  Some hidden props were passed from Gradebook component to PageButtons, which were removed during de-monolith-ization.  Adding them back in directly to the PageButtons component.

JIRA: [JIRA-5822](https://openedx.atlassian.net/browse/JIRA-5822)

**What changed?**

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**
- [x] Test suites passing
- [ ] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by matthugs — APPROVED** [review]

**Comment by jansenk** [comment]
Huzzah, thank you for doing this

**Review by jansenk — APPROVED** [review]
LGTM

**Review by leangseu-edx — APPROVED** [review]
